### PR TITLE
add citation

### PR DESCRIPTION
**Pull request by ZedongPeng**

### CODE REVIEW

**Comment by bernalde** [comment]
Use this citation file instead:
`@incollection{bernal2022advances,
  title={Advances in Generalized Disjunctive and Mixed-Integer Nonlinear Programming Algorithms and Software for Superstructure Optimization},
  author={Bernal, David E and Liu, Yunshan and Bynum, Michael L and Laird, Carl D and Siirola, John D and Grossmann, Ignacio E},
  booktitle={Computer Aided Chemical Engineering},
  volume={49},
  pages={1285--1290},
  year={2022},
  publisher={Elsevier}
}`

**Comment by ZedongPeng** [comment]
@bernalde

File: CITATION.ccf [hunk 1]
```diff
@@ -0,0 +1,38 @@
+cff-version: 1.0.0
+message: "If you use this software, please cite it as below."
+authors:
+  - family-names: Bernal
+    given-names: David E.
+    orcid: https://orcid.org/0000-0002-8308-5016
+  - family-names: Chen
+    given-names: Qi
+  - family-names: Liu
```
Comment (bernalde) [inline CITATION.ccf:13]: Add yourself @ZedongPeng and add my second last name. Also search if the other authors have orcid.

**Review by bernalde — CHANGES_REQUESTED** [review]

File: CITATION.ccf (hunk 1 above)
Comment (ZedongPeng) [reply CITATION.ccf:13]: done.

**Review by bernalde — APPROVED** [review]

**Pull request merged**